### PR TITLE
Adds file size sensor

### PR DESCRIPTION
**Pull request by robmarkcole**
**Description:**
Component for displaying the size of a file.

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant/pull/12211) (if applicable):** home-assistant/home-assistant#12211

## Checklist:

- [ ] Branch: Fixes, changes and adjustments should be created against `current`. New documentation for platforms/components and features should go to `next`.
- [ x] The documentation follow the [standards][standards].

[standards]: https://home-assistant.io/developers/documentation/standards/

### CODE REVIEW

File: source/_components/sensor.filesize.markdown [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+Add to your config:
+
+```yaml
+sensor:
+  - platform: filesize
+    file_paths:
+      - /config/home-assistant_v2.db
+  ```
+
+Configuration variables:
```
Comment (frenck) [inline source/_components/sensor.filesize.markdown]: Please use the configuration tags, for more information please see:
https://home-assistant.io/developers/documentation/create_page/#configuration

**Comment by frenck** [comment]
Thanks, @robmarkcole! 
Could you please take a look at the comment I've left?

File: source/_components/sensor.filesize.markdown (hunk 1 above)
Comment (robmarkcole) [reply source/_components/sensor.filesize.markdown]: Sure! Btw I don't think this rule is being applied consistently by reviewers? See [example](https://github.com/dgomes/home-assistant.github.io/blob/d9186d325e9c54c322a8bec9e29fea27220e3bbb/source/_components/sensor.sql.markdown)

Comment (frenck) [reply source/_components/sensor.filesize.markdown]: It is consistent. If you take a look at that PR, you would have seen my RFC for the same thing as well (which I did before you made the above comment).

Secondly, you need to remove the "Configuration variables:" header as well.

**Comment by frenck** [comment]
Can be merged as soon as the parent PR gets merged.

**Pull request merged**